targetplatform/EclipseSmartHome.setup

```diff
@@ -2376,7 +2376,7 @@
       <repository
           url="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/releases/"/>
       <repository
-          url="http://dist.springsource.org/snapshot/GRECLIPSE/e4.7/"/>
+          url="http://dist.springsource.org/snapshot/GRECLIPSE/e4.8"/>
```

Comment:
Does the p2 resolution work for Photon and Oxygen (e.g. also the Xtext / EMF stuff)?

Reply:
Yes, both versions are supported from the 4.8 snapshot repo (atm).